phis2-ws/src/main/java/phis2ws/service/dao/sesame/TraitDaoSesame.java

```diff
@@ -453,4 +460,39 @@ public POSTResultsReturn checkAndUpdate(List<TraitDTO> traitsDTO) {
             return checkResult;
         }
     }
+    
+    protected SPARQLQueryBuilder prepareSearchQueryVariables(Trait trait) {
```

Comment:
I have two remarks for this method : 
- **Documentation** : could you add the method documentation and move the query example in the documentation ? 
- **Search param** : in this case, we will always search the list of the variables for a given trait uri. I think that it would be better to send directly the traitUri. When the param is a Trait, it makes me think that the query could use all the trait informations to generates its query.
```suggestion
/**
 * ....
 * @example 
 * SELECT DISTINCT ?varUri
  * WHERE {
  * ?varUri <http://www.phenome-fppn.fr/vocabulary/2017#hasTrait> http://www.phenome-fppn.fr/platform/id/traits/t001 .}
  * @return the query generated.
 **/
    protected SPARQLQueryBuilder prepareSearchQueryVariables(String traitUri) {
```